drf_extra_fields/fields.py

```diff
@@ -30,6 +30,11 @@
 
 
 class Base64FieldMixin(object):
+
+    @property
+    def ALLOW_ALL_TYPES(self):
+        return False
```

Comment:
How about we refer to all types with `ALLOWED_TYPES = ["*"]`. Django also uses this approach. Please see  https://docs.djangoproject.com/en/2.2/ref/settings/#allowed-hosts

Then we can use `if self.ALLOWED_TYPES is not ["*"] and file_extension not in self.ALLOWED_TYPES:` 

That way there will be one way of controlling allowed types.

How that sounds?

Reply:
Sounds good, much better than then having two attributes controlling the same thing.